store/src/main/java/org/apache/rocketmq/store/queue/RocksDBConsumeQueueStore.java

```diff
@@ -500,6 +529,16 @@ public ConsumeQueueInterface findOrCreateConsumeQueue(String topic, int queueId)
         return oldLogic != null ? oldLogic : newLogic;
     }
 
+    @Override
+    public ConcurrentMap<Integer, ConsumeQueueInterface> findConsumeQueueMap(String topic) {
+        if (MixAll.isLmq(topic)) {
+            ConcurrentMap<Integer, ConsumeQueueInterface> result = new ConcurrentHashMap<>(1);
+            result.put(MixAll.LMQ_QUEUE_ID, findOrCreateConsumeQueue(topic, MixAll.LMQ_QUEUE_ID));
+            return result;
+        }
+        return super.findConsumeQueueMap(topic);
+    }
```

Comment:
I don't quite understand the necessity of overriding this method. Could you please clarify?

Reply:
This override SHOULD not be included in the pull request, as it makes sense internally only.